benchto-driver/src/main/java/com/teradata/benchto/driver/Benchmark.java

```diff
@@ -13,18 +13,18 @@
  */
 package com.teradata.benchto.driver;
 
-import com.facebook.presto.jdbc.internal.guava.collect.ImmutableMap;
 import com.google.common.base.Objects;
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
```

Comment:
can you please modiify checkstyle (https://github.com/prestodb/benchto/blob/master/src/checkstyle/checks.xml) to fail if `com...internal` is used in imports?